Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Builder
+FROM python:3.11 as builder
+
+# Note: this value currenlty needs to be the same as the value returned from get_version() from setup.py
+ARG ROOKIFY_VERSION=0.0.1
+
+WORKDIR /app/rookify
+
+RUN pip install build
+COPY . /app/rookify/
+RUN python -m build /app/rookify/
+
+
+# Base
+FROM debian:stable-slim as base
+
+COPY requirements.txt /app/rookify/src/
+
+# Install Librados:
+# Get rados version from requirements.txt and install it using apt,
+# because librados does not exist as pip package
+RUN apt-get update && apt-get install -y python3-pip python3-rados python3-venv && \
```

Comment:
I think we should build librados and python3-rados from source, so that we can enforce the librados/python-rados version to the required version

Reply:
will check if we can use a ceph image. Seems that building librados requires building a whole ceph.